packages/strapi-helper-plugin/lib/src/utils/auth.js

```diff
@@ -21,7 +21,12 @@ const auth = {
 
   clearAppStorage() {
     if (localStorage) {
+      const videos = auth.get('videos');
```

Comment:
Maybe we can pass the keys that we don't want to delete. But it will subject to another PR.